app/views/courses/gcse_requirements/edit.html.erb

```diff
@@ -38,8 +38,9 @@
             <% if @course.is_primary? %>
               <%= f.govuk_check_box :accept_science_gcse_equivalency, "Science", label: { text: "Science" }, data: { qa: "gcse_requirements__science_equivalency" } %>
             <% end %>
-            <%= f.govuk_text_area :additional_gcse_equivalencies, label: { text: "Details about equivalency tests you offer or accept", size: "s" }, hint: { text: "For example, if you offer the tests or ask candidates to use a third party, and if there are any costs to pay." }, data: { qa: "gcse_requirements__additional_requirements" } %>
+            <br>
```

Comment:
Not sure why this is here

```suggestion
```

Reply:
See https://github.com/DFE-Digital/publish-teacher-training/pull/1788#discussion_r664011653